### PR TITLE
Added a parameter for mesh pick up (onPick is for pick down)

### PR DESCRIPTION
**Pull request by Temechon**

### CODE REVIEW

File: Babylon/babylon.scene.ts [hunk 1]
```diff
@@ -465,6 +467,30 @@
...
+
+                if (!this.onPointerUp) {
+                    predicate = (mesh: AbstractMesh): boolean => {
+                        return mesh.isPickable && mesh.isVisible && mesh.isReady() && mesh.actionManager && mesh.actionManager.hasPickTriggers;
+                    };
+                }
+
+                this._updatePointerPosition(evt);
+
+                var pickResult = this.pick(this._pointerX, this._pointerY, predicate, false, this.cameraToUseForPointers);
```
Comment (deltakosh) [inline Babylon/babylon.scene.ts]: This should be done ONLY if pickup is defined

**Comment by Temechon** [comment]
I don't understand what you're trying to tell me :)

**Comment by Temechon** [comment]
Oh, you mean put the scene.pick inside the (if this.onPointerUp) block ?

**Comment by deltakosh** [comment]
You should not do a pick testing if onPickup is not defined by the user

File: Babylon/babylon.scene.ts (hunk 1 above)
Comment (deltakosh) [inline Babylon/babylon.scene.ts]: Nevermind, I'm saying shit:) I should not talk when I'm tired

Comment (deltakosh) [inline Babylon/babylon.scene.ts]: I'll validate it asap

Comment (Temechon) [inline Babylon/babylon.scene.ts]: Cool, thanks :) Take your time ;)

**Pull request merged**